### PR TITLE
Drivetrain

### PR DESCRIPTION
**Pull request by simon-andrews**
Seems like drivetrain work has slowed down, so this can go into master.

### CODE REVIEW

**Review by bshin100 — DISMISSED** [review]
👍

File: src/org/robockets/steamworks/commands/ToggleDriveMode.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * A command to toggle a boolean used in Joyride.java for switching drive mode.
+ * @author Simon Andrews & Brian Shin
+ */
+public class ToggleDriveMode extends Command {
+
+	public static boolean isArcade;
```
Comment (jakebacker) [inline src/org/robockets/steamworks/commands/ToggleDriveMode.java]: Weird indentation

File: src/org/robockets/steamworks/commands/Joyride.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    protected void initialize() {
+    }
+
+    protected void execute() {
+    	if(ToggleDriveMode.isArcade) {
+    		//RobotMap.robotDrive.arcadeDrive(OI.joystick, 1, OI.joystick, 4);
+    		RobotMap.robotDrive.arcadeDrive(OI.joystick.getRawAxis(1), OI.joystick.getRawAxis(4));
+    		OI.joystick.setRumble(RumbleType.kRightRumble, 0.0);
+    	}
+    	else {
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/commands/Joyride.java]: WHYYYY!

File: src/org/robockets/steamworks/commands/Joyride.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.steamworks.OI;
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/commands/Joyride.java:6]: New line

File: src/org/robockets/steamworks/Robot.java [hunk 4]
```diff
@@ -48,19 +51,24 @@ public void robotInit() {
...
+		conveyor = new Conveyor();
     climber = new Climber();
-    conveyor = new Conveyor();
 		drivetrain = new Drivetrain();
 		shooter = new Shooter();
 
 		// chooser.addObject("My Auto", new MyAutoCommand());
 		climb = new Climb(0.5);
-    drive = new GottaGoFast(0.5);
+    drive = new Joyride(0.5);
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/Robot.java]: weird indentation

File: src/org/robockets/steamworks/Robot.java [hunk 5]
```diff
@@ -48,19 +51,24 @@ public void robotInit() {
 		oi = new OI();
 
 		ballIntake = new BallIntake();
+		conveyor = new Conveyor();
     climber = new Climber();
```
Comment (UnknownSilicon) [inline src/org/robockets/steamworks/Robot.java]: weird indentation

**Review by UnknownSilicon — CHANGES_REQUESTED** [review]
👎

**Review by UnknownSilicon — APPROVED** [review]

**Comment by UnknownSilicon** [comment]
Tabs>Spaces
Spaces.isReal = false

**Review by bshin100 — APPROVED** [review]

**Pull request merged**